v2/client/src/components/pages/survey/Survey.js

```diff
@@ -45,10 +39,28 @@ export default class Survey extends React.Component {
         });
       })
       .then(() => {
-        swal.fire({
-          text:
-            'Many thanks for agreeing to fill in this form. Your feedback will be kept anonymous and will only take you a couple of minutes to complete. Your feedback helps us evaluate and improve the program.',
-        });
+        swal
+          .fire({
+            title: 'Research Cooperation <br>(University of Manchester)',
+            text:
+              'Many thanks for agreeing to fill in this form. Your responses will be collated by University of Manchester to evaluate Connect5. University of Manchester will use anonymised data collected for research purposes. Individuals will never been identified by their responses. If you do not consent for your data to be used for research purposes, please tick.',
```

Comment:
Just an idea:

better group all the text together in one file, so that when you need to make changes later on (or add translations for text), it is easier to maintain and without jumping around in the code base to search for it